integrations/viborg/viborg.py

```diff
@@ -75,7 +74,7 @@
     manager_rows=manager_rows
 )
 
-sd.create_ou_tree()
+sd.create_ou_tree(create_orphan_container=False)
```

Comment:
Vil vi nu ikke have den container alligevel?

Reply:
I Viborg er det såleds at de faktisk ikke har nogle forældreløse enheder, og derfor er der ikke behov for at oprette denne container.